regression/Makefile

```diff
@@ -25,6 +25,7 @@ DIRS = cbmc \
        goto-cc-goto-analyzer \
        systemc \
        contracts \
+       goto-cc-file-local \
```

Comment:
Don't forget CMake!

Reply:
Well, the message on the past PR should have been "Don't forget Make" :-) - CMakeLists.txt already had it...